policy/modules/system/logging.te

```diff
@@ -548,6 +548,16 @@ ifdef(`init_systemd',`
 	systemd_manage_journal_files(syslogd_t)
 
 	udev_read_runtime_files(syslogd_t)
+
+	# needed for systemd --user instances
+	dontaudit syslogd_t self:capability dac_read_search;
+
```

Comment:
This is so that journald can traverse /run/user/* to get to  /run/user/*/systemd/units. Why dontaudit that it?

Reply:
In my testing I found that even if this permission is not allowed, it still works, though I don't see much harm in allowing this. Your thoughts?

Reply:
hmm i send a reply to this via email but seems it has not arrived here. maybe its still in transit....

Reply:
The gist is this:

/run/user/* is more 0700 and so journald needs cap_dac_read_search to traverse it so that it can get to /run/user/*/systemd/units where it needs only to read symlinks.

journald AFAIK should not be reading systemd_user_runtime_t files

dontaudits should not be needed

Reply:
OK I understand now. Fixing this and the below.

Reply:
if your going to comment that rule that atleast use a comment that is helpful:

# journald traverses /run/user/UID (which is mode 0700) to read symlinks in /run/user/UID/systemd/units/